tidb-server/main.go

```diff
@@ -93,6 +94,11 @@ var (
 	metricsAddr     = flag.String(nmMetricsAddr, "", "prometheus pushgateway address, leaves it empty will disable prometheus push.")
 	metricsInterval = flag.Int(nmMetricsInterval, 15, "prometheus client push interval in second, set \"0\" to disable prometheus push.")
 
+	//TODO: enablePreparedPlanCache parameter will be added to session or global variables so that can enable the plan cache without restarting their service at any time
+	enablePreparedPlanCache = flagBoolean("enable-plan-cache-for-prepare", true, "use plan cache for prepared statements")
```

Comment:
how about move these flags to `config/config.go` and `config/config.toml.example` ?

Reply:
OK, I'll handle it another PR.